src/helpers/channel/types.ts

```diff
@@ -1,67 +1,73 @@
 export type ChannelSetting =
-| {
-    type: 1; // Boolean
-    default: boolean;
-    description: string;
-    index: number;
-  }
-| {
-    type: 2; // Range
-    default: number;
-    enabled: boolean;
-    description: string;
-    index: number;
-    lowerLimit: number;
-    upperLimit: number;
-    ticker: number;
-  }
-| {
-  type: 3; // Range
-  default: {
-    lower: number;
-    upper: number;
-  };
-  enabled: boolean;
-  description: string;
-  index: number;
-  lowerLimit: number;
-  upperLimit: number;
-  ticker: number;
-};
+  | {
+      type: 1; // Boolean
+      default: boolean;
+      description: string;
+      index: number;
+    }
+  | {
+      type: 2; // Range
+      default: number;
+      enabled: boolean;
+      description: string;
+      index: number;
+      lowerLimit: number;
+      upperLimit: number;
+      ticker: number;
+    }
+  | {
+      type: 3; // Range
+      default: {
+        lower: number;
+        upper: number;
+      };
+      enabled: boolean;
+      description: string;
+      index: number;
+      lowerLimit: number;
+      upperLimit: number;
+      ticker: number;
+    };
 
 export type UserSetting =
-| {
-    type: 1; // Boolean
-    default: boolean;
-    description: string;
-    index: number;
-    user: boolean;
-  }
-| {
-    type: 2; // Range
-    default: number;
-    enabled: boolean;
-    description: string;
-    index: number;
-    lowerLimit: number;
-    upperLimit: number;
-    user: number;
-    ticker: number;
-  }
-| {
-  type: 3; // Range
-  default: {
-    lower: number;
-    upper: number;
-  };
-  enabled: boolean;
-  description: string;
-  index: number;
-  lowerLimit: number;
-  upperLimit: number;
-  user: {
-    lower: number;
-    upper: number;
-  };
-  ticker: number;
-};
+  | {
+      type: 1; // Boolean
+      default: boolean;
+      description: string;
+      index: number;
+      user: boolean;
+    }
+  | {
+      type: 2; // Range
+      default: number;
+      enabled: boolean;
+      description: string;
+      index: number;
+      lowerLimit: number;
+      upperLimit: number;
+      user: number;
+      ticker: number;
+    }
+  | {
+      type: 3; // Range
+      default: {
+        lower: number;
+        upper: number;
+      };
+      enabled: boolean;
+      description: string;
+      index: number;
+      lowerLimit: number;
+      upperLimit: number;
+      user: {
+        lower: number;
+        upper: number;
+      };
+      ticker: number;
+    };
+
+export enum NOTIF_PERMISSION_STATUS {
```

Comment:
`export type NotificationPermissionStatus  = "granted" | "denied" | "pending" | "notValid";`

Reply:
why is this needed @rohitmalhotra1420